kornia/core/check.py

```diff
@@ -459,3 +454,13 @@ def KORNIA_CHECK_LAF(laf: Tensor, raises: bool = True) -> bool:
 
     """
     return KORNIA_CHECK_SHAPE(laf, ["B", "N", "2", "3"], raises)
+
+
+def _handle_invalid_range(msg: Optional[str], raises: bool, min_val, max_val) -> bool:
```

Comment:
@johnnv1 what type annotation do you recommend i use here for min_val and max_val?

Reply:
float | int might suffice i think